src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
```

Comment:
@ctiller, @jtattermusch: would this method better match the capabilities delivered elsewhere in gRPC if it were changed to offer one-parameter-per-propagateable-thing? As in:
```
def association(
    self, deadline=None, census_stats=None, census_tracing=None,
    cancellation=None):
```
(that parameter order comes from [this order](https://github.com/grpc/grpc/blob/01d7dbac1f747ca45135f1d79c38656db0701ec6/include/grpc/impl/codegen/propagation_bits.h#L46); feel invited to request a better one).